packages/rich-text/src/test/performance/rich-text.native.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import {
+	changeTextOfRichText,
+	fireEvent,
+	measurePerformance,
+	screen,
+} from 'test/helpers';
+
+/**
+ * Internal dependencies
+ */
+import RichText from '../../component/index.native';
+
+describe( 'RichText Performance', () => {
+	const onCreateUndoLevel = jest.fn();
+	const onChange = jest.fn();
+	const onSelectionChange = jest.fn();
+
+	it( 'performance is stable when typing using Rich Text', async () => {
```

Comment:
We discussed and changed the description to avoid duplication at one point. I am uncertain if it was intentionally changed back. If it was intentional, feel free to ignore this. 
```suggestion
	it( 'is stable when typing', async () => {
```